docs/modules/README.md

```diff
@@ -14,6 +14,8 @@ ESLint plugin.
 
 - [`@jsdevtools/ez-spawn`](./process-exec.md)
 - [`bluebird` / `q`](./bluebird-q.md)
+- [`body-parser`](./body-parser.md)
+- [`builtin-modules`](./is-builtin-module.md)
```

Comment:
Did you mean to add this line?

Was it missing before?

Reply:
Yes, they're surprisingly not available before.

Reply:
ah it exists u nder `is-builtin-module` here:

https://github.com/es-tooling/module-replacements/blob/7d864c64f68394ea55f33ef67c712c217e5e68c4/docs/modules/README.md?plain=1#L30

Reply:
Oops, I missed that since I thought the entries were lexicographically sorted by the markdown name. Removed.